packages/office-ui-fabric-react/src/components/OverflowSet/examples/OverflowSet.Custom.Example.tsx

```diff
@@ -12,9 +10,9 @@ export class OverflowSetCustomExample extends React.PureComponent {
         aria-label="Custom Example"
         items={[
           {
-            key: 'search',
+            key: 'checkbox',
             onRender: () => {
-              return <SearchBox placeholder="Search" styles={{ root: { marginBottom: 0, width: 200 } }} />;
+              return <Checkbox label="A Checkbox" styles={{ root: { marginRight: 5 } }} />;
```

Comment:
Looks like searchbox doesn't do well with narrator scan mode so switching to checkbox instead?